### ISSUE TITLE
Debug slowness

### ISSUE THREAD

**Issue by baopham** [gregsexton/gitv#125]
How do I debug when an action takes longer than expected? Currently, when I do a `D` in diff mode, the CPU goes wild.

**Comment by rbong** [comment]
Usually, what I do is

```
:redir @a | 20verbose exec 'normal D' | redir END
:tabe
"ap
```

Hope that helps. Suggestions for built in debugging utilities is welcome.

**Comment by baopham** [comment]
Thank you for the snippet. I couldn't get much info 🤔  I think it could be the git command itself but don't think currently there's a way we can print out the git command being executed?

**Comment by rbong** [comment]
No, there isn't a way to currently do that unfortunately. Better error handling is planned.
